pkg/question/question.go

```diff
@@ -719,22 +773,25 @@ func AskSubnetPlaceholder(h *ec2helper.EC2Helper) (*string, error) {
 	indexedOptions := []string{}
 
 	// Add availability zones to the data for table
+	var defaultOptionValue *string
 	for index, zone := range availabilityZones {
+		if userDefaultAzId != "" && *zone.ZoneId == userDefaultAzId {
+			defaultOptionValue = zone.ZoneName
```

Comment:
Is this going to get overwritten on line 785? I remember you noticed a bug with subnet AZ selection during testing--is this where that manifests?